app/src/main/java/org/oppia/app/databinding/GuidelineBinding.kt

```diff
@@ -0,0 +1,12 @@
+package org.oppia.app.databinding
+
+import androidx.constraintlayout.widget.ConstraintLayout
+import androidx.constraintlayout.widget.Guideline
+import androidx.databinding.BindingAdapter
+
+@BindingAdapter("app:layout_constraintGuide_percent")
```

Comment:
Add kdoc for this. Also, if you check other binding adapters, you should follow this convention,

`layoutConstraitGuidePercent`

Reply:
> if you check other binding adapters, you should follow this convention,
`layoutConstraitGuidePercent`

I named it `app:layout_constraintGuide_percent` to override the existing property in the app namespace, I'm not creating a new one here.

Reply:
>Add kdoc for this.

Done